docs/conf.py

```diff
@@ -98,18 +131,12 @@
 ]
 exclude_patterns.extend(custom_excludes)
 
-rst_epilog = '''
-.. include:: /reuse/links.txt
```

Comment:
Remove default reusable links that we don't use from the starter pack, so that we can have one less static file in our docs folder.